.github/workflows/tag_image_push.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        jdk: [ '11', '17', '21' ]
```

Comment:
We should use jdk variable in DOCKER_LOG_FILE_OSS and DOCKER_LOG_FILE_EE

Reply:
done